inc/auth.php

```diff
@@ -1354,6 +1361,10 @@ function auth_cryptPassword($clear, $method = '', $salt = null)
  */
 function auth_verifyPassword($clear, $crypt)
 {
+    if ($crypt === UNUSABLE_PASSWORD) {
```

Comment:
This check could probably be omitted since the string "!unusable" will never match any known hashing algo?

Reply:
I don't think so either, but I still would leave it here. Better safe than sorry.